### PR TITLE
onramps style guide

### PR DESCRIPTION
**Pull request by ghost**
https://issues.newrelic.com/browse/NR-111984
https://docs.google.com/document/d/1fzxzX7l578Jm0uiWF9iQGeSciAipPz25Nk1SrmwuKMc/edit

### CODE REVIEW

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+You can get notifications through email, Slack, Jira, and more. To learn how to set up notifications, see [Introduction to alert notifications](/docs/alerts-applied-intelligence/notifications/intro-notifications/).
+
+### Regular headers
+
+If FAQ format is not applicable, use our standard headers for titles.
+
+Example:
+
+**Forward logs to New Relic**
+Forwarding logs to New Relic allows you to have all logs in one place.
```
Comment (ally-sassman) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:38]: @zuluecho9 Do you have any examples of onramps that's not in a question format? I just added this as placeholder.

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+## What if a tile has two states (active and empty)?
+
+A tile is something that's clickable and may or may not contain data, depending on if customers have set it up. This is called an active vs. an empty state, and it's something to consider when writing onramps content. If the tile has no data, it's in empty state, and if the user clicks on it, they may be directed to a set up page. If it has data, it's in active state, and our onramps content will want to help users interpret the data.
+
+For example, here's the Incidents tile in active and empty states, and the content that touches on both scenarios.
```
Comment (ally-sassman) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:44]:
```suggestion
For example, here's a tile in active and empty states, and the content that touches on both scenarios.
```

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+import apmActiveEmptyState from 'images/apm_screenshot-crop_active-empty-state.webp'
+
+Onramps content is information that appears our UI when a user clicks the **?** icon. This content is meant to be truncated and contextualized to where the user is and what they need to know to perform a task.
+
+## How should I name the section titles?
+
+When applicable, use an FAQ format where the headers are in question format.
+
+For example:
```
Comment (ally-sassman) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:19]:
```suggestion
For example, here's the Issues tile with headers formatted into questions.
```

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+import apmActiveEmptyState from 'images/apm_screenshot-crop_active-empty-state.webp'
+
+Onramps content is information that appears our UI when a user clicks the **?** icon. This content is meant to be truncated and contextualized to where the user is and what they need to know to perform a task.
+
+## How should I name the section titles?
+
+When applicable, use an FAQ format where the headers are in question format.
+
+For example, here's the Issues tile with headers formatted into questions.
```
Comment (jeff-colucci) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:19]: Should issues be bolded? I'm not actually sure myself so I figure it can't hurt to ask!

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    src={apmOnrampsIssuesExample}
+    style={{ maxWidth: '50%' }}
+/>
+
+
+If FAQ format is not applicable, use our standard headers for titles.
+
+## What if a tile has two states (active and empty)?
+
+A tile is something that's clickable and may or may not contain data, depending on if customers have set it up. This is called an active vs. an empty state, and it's something to consider when writing onramps content. If the tile has no data, it's in empty state, and if the user clicks on it, they may be directed to a set up page. If it has data, it's in active state, and our onramps content will want to help users interpret the data.
```
Comment (jeff-colucci) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:33]: I always found "or may not" to be covered by "may", but it's not a big deal.

Comment (jeff-colucci) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:33]: Should there be some differentiator for "active" and "empty" state?

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    alt=""
+    src={apmActiveEmptyState}
+    style={{ maxWidth: '75%' }}
+/>
+
+## How long should the content be?
+
+If you're not using collapsers, **keep the character count under 1200** for the entire onramps section.
+
+## How should I bold/italicize to emphasize something?
```
Comment (jeff-colucci) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:48]: I think you can cut the "how" here? It seems that the subsection is covering whether or not you should bold/italicize for emphasis, not how to actually do it.

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+Since we can't add code formatting to onramps content, use bold and italics to emphasize something as needed.
+
+## How much detail should I include?
+
+If the onramps content isn't in the main place where the feature is (i.e., onramps content for the Service levels tile on the APM Summary page), then it’s not the place to include a lot of detail.
```
Comment (jeff-colucci) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:54]: I think APM Summary should be bolded.

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+## Should I add links out to docs?
```
Comment (jeff-colucci) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:56]: Links out is makes my brain stop for a second. Maybe "Should I link to other docs?" might be easier to parse? Just a thought.

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+
+It's okay to add a link to docs "for more details" on something, but here are some considerations:
+* Add a docs link if there's relevant information and it's too long to summarize/explain in onramps. 
+* The onramps content will persist after the user clicks on something, so add a docs link if it's relevant to where the user might go next (i.e., if you click on the Issues tile and are taken to the **Issues** page, you'll still see the onramps popup on the same, so you may want to learn how to configure issue thresholds on this page).
+
+## How should I reference UI navigation instructions?
+
+ Let's say you want to direct users to something relevant in the UI and need to add navigation instructions. For example, if a user clicks on the Issues tile and they want to get alerts for issues (which is on another UI page), you can provide instructions in two ways:
```
Comment (jeff-colucci) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:64]: Like elsewhere, should Issues be bolded?

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+ * Link them to the docs (i.e., "To create an alert condition, see [Create your first alert](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition/).")
```
Comment (jeff-colucci) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:65]: I think the link formatting is wrong here with the https: part.

**Review by jeff-colucci — DISMISSED** [review]
Looks good to me! I only saw minor things, nothing gating, so you've got my approval!

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+
+## How much detail should I include?
+
+If the onramps content isn't in the main place where the feature is (i.e., onramps content for the Service levels tile on the APM Summary page), then it’s not the place to include a lot of detail.
+
+## Should I add links out to docs?
+
+It's okay to add a link to docs "for more details" on something, but here are some considerations:
+* Add a docs link if there's relevant information and it's too long to summarize/explain in onramps. 
+* The onramps content will persist after the user clicks on something, so add a docs link if it's relevant to where the user might go next (i.e., if you click on the Issues tile and are taken to the **Issues** page, you'll still see the onramps popup on the same, so you may want to learn how to configure issue thresholds on this page).
```
Comment (mmfred) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:60]:
```suggestion
* The onramps content will persist after the user clicks on something, so add a docs link if it's relevant to where the user might go next (i.e., if you click on the Issues tile and are taken to the **Issues** page, you'll still see the onramps popup on the same page, so you may want to learn how to configure issue thresholds on this page).

Avoid making a link to docs the sole content of an onramp.  When linking, use the established UI style for docs links. Text is "See our docs" and use the external link icon. 
```

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx (hunk 10 above)
Comment (mmfred) [reply src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:65]: This one takes them out of the product, so I wouldn't put it first. And is it using a link to docs as shorthand for more info? That's why I recommend that a link to docs not be the only content in an onramp.

**Review by mmfred — COMMENTED** [review]
I'll follow up in Slack.

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+contentType: page
+template: basicDoc
+topics:
+  - onramps
+---
+
+import apmOnrampsIssuesExample from 'images/apm_screenshot-crop_onramps-issues-example.webp'
+
+import apmActiveEmptyState from 'images/apm_screenshot-crop_active-empty-state.webp'
+
```
Comment (paperclypse) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:12]: @ally-sassman It would be great to add a section in this style guide doc about the purposes of on-ramps content:
1. We want to enable users to get more value out of the UI
2. We want to help provide a larger context for why and how people can use these pieces of the UI to solve their problems and connect them to other parts of the UI
3. We want to provide info on how users can solve specific problems with the part of the UI they're looking at.

Comment (paperclypse) [reply src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:12]: Feel free to chat with me about this if you want to talk through it.

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx (hunk 11 above)
Comment (zuluecho9) [reply src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:60]: regarding Michelle's comment above, the external link icon she mentions works automatically for links so we don't have to mention that apparently.

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx (hunk 10 above)
Comment (zuluecho9) [reply src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:65]: and reminder about Michelle's point; we apparently want to follow the UI writing style of only using 'see our docs' as the only link text.

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+Onramps content is information that appears our UI when a user clicks the **?** icon. This content is meant to be truncated and contextualized to where the user is and what they need to know to perform a task.
+
+## How should I name the section titles?
```
Comment (zuluecho9) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:15]: Think we should say that headers can be any format, as long as they're short/succinct. We have quite a few that are questions, and we have quite a few that are not questions, and maybe we specifically say there is no rule around that.

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+<img
+    title=""
+    alt=""
+    src={apmActiveEmptyState}
+    style={{ maxWidth: '75%' }}
+/>
+
+## How long should the content be?
+
+If you're not using collapsers, **keep the character count under 1200** for the entire onramps section.
```
Comment (zuluecho9) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:46]: did we want to add a rule for limiting a sub-section of an onramp?

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx (hunk 12 above)
Comment (ally-sassman) [reply src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:12]: **Notes**

The purpose is to help people with the thing they're looking at when it's not obvious.

The most important questions are the why and the how.

However, we want the UI to speak for itself, so we only use it when there's ambiguity. 

Users also use it to solve a problem (i.e., why is the Apdex page important to me?) 

There's a larger context - how does a piece of the UI fit into the larger context? I.e., how do APM and logs relate to each other? How does APM and infrastructure relate to either other? Here's how this aspect of APM relates to this piece of infrastructure. How are Issues related to Alerts? 

The secondary goal is if there's ambiguity/something's not clearly defined, we can take action as the docs team to provide a patchwork of sorts, filling in where there may be confusion.

The business goal is that in-product help will help users get more out of NR, and in turn we make more money.

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 15]
```diff
@@ -0,0 +1,105 @@
...
+
+import apmActiveStateExample from 'images/apm_screenshot-crop_active-state-example.webp'
+
+import apmServiceLevelsExample from 'images/apm_screenshot-crop_service-levels-example.webp'
+
+
+Our onramps content is the help text that appears in the UI when you click the **?** icon. The purpose of in-product help is to help users understand the UI, interpret performance data, and New Relic solutions with their system problems. Onramps provide more detail than a [tooltip](https://one-core.newrelic.com/components/tooltip/). Ideally, they give users essential info so they can stay in the product and keep working, rather than switching over to read the docs. When writing onramps, include information that helps users:
+- Understand what a feature is used for, and the benefits of the feature beyond the existing heading or other content. 
+- Perform specific tasks.
+- Understand what to do next. Are there recommended follow-on actions? The Apdex troubleshooting onramp provides a nice example for this. It tells users: "Compare the Apdex chart with other charts (like Web transaction time and Error rate) to look for the causes for the Apdex change."
```
Comment (ally-sassman) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:21]:
```suggestion
* Understand what a feature is used for, and the benefits of the feature beyond the existing heading or other content. 
* Perform specific tasks.
* Understand what to do next. Are there recommended follow-on actions? The Apdex troubleshooting onramp provides a nice example for this. It tells users: "Compare the Apdex chart with other charts (like Web transaction time and Error rate) to look for the causes for the Apdex change."
```

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx (hunk 14 above)
Comment (ally-sassman) [reply src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:90]: @paperclypse What do you think? My thought against adding it is it feels subjective to what the onramps is about. What if the onramp only needs one or two sections (vs. 3-5)? Then the sections could be longer if there are only 1-2, and s section limit wouldn't always be applicable to all onramps.

Comment (paperclypse) [reply src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:90]: I hesitate to add hard and fast rules like this... But maybe we should rephrase this as: Abide by these limits, unless you have a really good reason not to.

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx (hunk 13 above)
Comment (mmfred) [reply src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx]: Recommendation for heading: make sure it's clear what the heading relates to. For example, don't say, "What is this?" -- instead say "What's an issue?" This is important because the info is off the side of the page, and if you change the page the onramps stay open.

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 16]
```diff
@@ -0,0 +1,115 @@
...
+
+Our onramps content is the help text that appears in the UI when you click the **?** icon. The purpose of in-product help is to help users understand the UI, interpret performance data, and New Relic solutions with their system problems. Onramps provide more detail than a [tooltip](https://one-core.newrelic.com/components/tooltip/). Ideally, they give users essential info so they can stay in the product and keep working, rather than switching over to read the docs. 
+
+<img
+    title=""
+    alt=""
+    src={apmApmSummaryOnrampsGif}
+/>
+
+## Content guidelines
```
Comment (austin-schaefer) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:28]: To me the entire doc reads like "Content guidelines." This section is about the heart of the content and what makes it really sing, whereas the rest of the doc is more like rules. I'd shoot for a heading that acknowledges that. What about something like, "What does a great on-ramp look like" or "Writing a great on-ramp" or similar.

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 17]
```diff
@@ -0,0 +1,115 @@
...
+
+* Be clear and concise. Contextual help should be clear and concise. Avoid using jargon or technical terms that users may not understand.
```
Comment (austin-schaefer) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:30]: Here and subsequently, I would bold the main idea to make these bullets easier to scan. (`**Be clear and concise.**`)

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 18]
```diff
@@ -0,0 +1,115 @@
...
+* Use second-person point of view to address the reader. Address the user as "you", such as "We couldn't find your data." 
+* Answer the "why" and "how" questions. For example, a user might ask themselves, "Why is this important to me?" or "How do I use this information to improve my app performance?" Write content to help answer these questions, to understand the larger context of their application. For example, a user can learn about how APM and logs relate to each other, or how APM and infrastructure relate to each other.
+* Provide additional context to clarify ambiguous or confusing areas in the product. For example, if there is something that is not clearly defined in the UI, we can provide additional information to help users understand it.
+* Help users get more out of New Relic (and increase our revenue). When users are able to use New Relic more effectively, they are more likely to renew their subscriptions and recommend us to others. This, in turn, helps us to make more money. Include information that helps users:
```
Comment (austin-schaefer) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:34]: I think we talked about this in the sprint review, but let's reframe these bullets a bit

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 19]
```diff
@@ -0,0 +1,115 @@
...
+  * Understand what a feature is used for, and the benefits of the feature beyond the existing heading or other content. 
+  * Perform specific tasks.
+  * Understand what to do next. Are there recommended follow-on actions? The Apdex troubleshooting onramp provides a nice example for this. It tells users: "Compare the Apdex chart with other charts (like Web transaction time and Error rate) to look for the causes for the Apdex change."
+
+## FAQs
```
Comment (austin-schaefer) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:39]: I'd like us to not use the word FAQ in our docs, including in style guide.

In terms of restructuring, I think you have two types of content a bit mixed under this FAQ:

* Rules-type content:bold/italics
    * On-ramp titles
    * Length
    * Bold/italics
    * Links
    * Tile states
    * UI navigation info
* More philosophical info, which may actually belong above with the other philosphical info
   * When is onramps contgent needed?
   * Level of detail

A few ideas for how to restructure it:

* Just lose the FAQ header and make the questions all h2s
* Move the philosphical ones up into the earlier section, and group the others together here under a Rules-type heading
* Move all of them into a table to make the structure a bit more skimmable

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 20]
```diff
@@ -0,0 +1,115 @@
...
+
+import apmActiveStateExample from 'images/apm_screenshot-crop_active-state-example.webp'
+
+import apmServiceLevelsExample from 'images/apm_screenshot-crop_service-levels-example.webp'
+
+
+Our onramps content is the help text that appears in the UI when you click the **?** icon. The purpose of in-product help is to help users understand the UI, interpret performance data, and New Relic solutions with their system problems. Onramps provide more detail than a [tooltip](https://one-core.newrelic.com/components/tooltip/). Ideally, they give users essential info so they can stay in the product and keep working, rather than switching over to read the docs. 
+
+<img
+    title=""
```
Comment (austin-schaefer) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:23]: Missing title and alt text (although not a dealbreaker on an internal-only doc like this)

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 21]
```diff
@@ -0,0 +1,115 @@
...
+
+If you're not using collapsers, **keep the character count under 1200** for the entire onramps section.
+
+### Should I bold/italicize to emphasize something?
+
+Since we can't add code formatting to onramps content, use bold and italics to emphasize something as needed.
+
+### How much detail should I include?
+
+If the onramps content isn't in the main place where the feature is (i.e., onramps content for the **Service levels** tile on the **APM Summary** page), then it’s not the place to include a lot of detail.
```
Comment (austin-schaefer) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:98]: Avoid `i.e.` and `e.g.` (a few instances of `i.e.` in this doc)

https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#eg

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 22]
```diff
@@ -0,0 +1,115 @@
...
+
+### Should I add links to docs?
+
+It's okay to add a link to docs for more details on something, but here are some considerations:
+* Add a docs link if there's relevant information and it's too long to summarize/explain in onramps. 
+* The onramps content will persist after the user clicks on something, so add a docs link if it's relevant to where the user might go next (i.e., if you click on the **Issues** tile and are taken to the **Issues** page, you'll still see the onramps popup on the same page, so you may want to learn how to configure issue thresholds on this page).
+
+Avoid making a link to docs the sole content of an onramp.
+
+When linking, use the established UI style for docs links. Use "See our docs" for the text.
```
Comment (austin-schaefer) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:108]: Generally we use code backticks rather than quotes in the style guide, although I don't think this is written down. See usage dictionary for some examples

**Review by austin-schaefer — DISMISSED** [review]

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 23]
```diff
@@ -0,0 +1,115 @@
...
+import apmOnrampsIssuesExample from 'images/apm_screenshot-crop_onramps-issues-example.webp'
+
+import apmEmptyStateExample from 'images/apm_screenshot-crop_empty-state-example.webp'
+
+import apmActiveStateExample from 'images/apm_screenshot-crop_active-state-example.webp'
+
+import apmServiceLevelsExample from 'images/apm_screenshot-crop_service-levels-example.webp'
+
+
+Our onramps content is the help text that appears in the UI when you click the **?** icon. The purpose of in-product help is to help users understand the UI, interpret performance data, and New Relic solutions with their system problems. Onramps provide more detail than a [tooltip](https://one-core.newrelic.com/components/tooltip/). Ideally, they give users essential info so they can stay in the product and keep working, rather than switching over to read the docs.
```
Comment (ally-sassman) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:20]:
```suggestion
Our in-product help content provides users with the tools they need to understand the New Relic UI, interpret performance data, and learn how new relic solutions can help with their problems. Onramps provide more detail than a [tooltip](https://one-core.newrelic.com/components/tooltip/). Ideally, they give users essential info so they can stay in the product and keep working, rather than switching over to read the docs. 
```

File: src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx [hunk 24]
```diff
@@ -0,0 +1,115 @@
...
+import apmServiceLevelsExample from 'images/apm_screenshot-crop_service-levels-example.webp'
+
+
+Our onramps content is the help text that appears in the UI when you click the **?** icon. The purpose of in-product help is to help users understand the UI, interpret performance data, and New Relic solutions with their system problems. Onramps provide more detail than a [tooltip](https://one-core.newrelic.com/components/tooltip/). Ideally, they give users essential info so they can stay in the product and keep working, rather than switching over to read the docs. 
+
+<img
+    title=""
+    alt=""
+    src={apmApmSummaryOnrampsGif}
+/>
```
Comment (ally-sassman) [inline src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx:26]:
```suggestion
<img
    title="Onramps example"
    alt="gif of onramps example in the UI"
    src={apmApmSummaryOnrampsGif}
/>
```

File: src/content/docs/style-guide/writing-docs/writer-workflow/onramps-style-guide.mdx [hunk 25]
```diff
@@ -0,0 +1,114 @@
...
+* **Provide additional context to clarify ambiguous or confusing areas in the product.** For example, if there is something that is not clearly defined in the UI, we can provide additional information to help users understand it.
+* **Help users get more out of New Relic.** Help users get the most out of New Relic. When users are able to use New Relic effectively, they are more likely to continue using our product and recommend us to others. Include information that helps users:
+  * Understand what a feature is used for, and the benefits of the feature beyond the existing heading or other content. 
+  * Perform specific tasks.
+  * Understand what to do next. Are there recommended follow-on actions? For example, the Apdex troubleshooting onramp provides a nice example for this. It tells users to `Compare the Apdex chart with other charts (like Web transaction time and Error rate) to look for the causes for the Apdex change.`
+
+## More guidelines
```
Comment (ally-sassman) [inline src/content/docs/style-guide/writing-docs/writer-workflow/onramps-style-guide.mdx:39]:
```suggestion
```

File: src/content/docs/style-guide/writing-docs/writer-workflow/onramps-style-guide.mdx [hunk 26]
```diff
@@ -0,0 +1,113 @@
...
+Our in-product help content provides users with the tools they need to understand the New Relic UI, interpret performance data, and learn how our solutions can help with their problems. Onramps provide more detail than a [tooltip](https://one-core.newrelic.com/components/tooltip/). Ideally, they give users essential info so they can stay in the product and keep working, rather than switching over to read the docs.
+
+<img
+    title="Onramps example"
+    alt="gif of onramps example in the UI"
+    src={apmApmSummaryOnrampsGif}
+/>
+
+
+## What a great onramp looks like
```
Comment (ally-sassman) [inline src/content/docs/style-guide/writing-docs/writer-workflow/onramps-style-guide.mdx:26]:
```suggestion
## Content guidelines
```
